cg/services/slurm_upload_service/slurm_upload_service.py

```diff
@@ -36,6 +36,10 @@ def upload(self, upload_command: str, job_name: str, case_id: str):
         slurm_id: int = self.slurm_service.submit_job(job_config)
         self.trailblazer_api.add_upload_job_to_analysis(slurm_id=slurm_id, analysis_id=analysis.id)
 
+    def add_upload_job_to_analysis(self, slurm_id: int, case_id: int) -> None:
+        analysis: TrailblazerAnalysis = self.trailblazer_api.get_latest_completed_analysis(case_id)
```

Comment:
Does this assumption hold? That the most recent analysis for a case is the analysis we want to associate the upload job with?

Reply:
Yes, I see some edge-cases where that might not be true but it's the best way we have

Reply:
Alright 👍